src/recognizers_store.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+package types;
+
+// PatternRecognizer is the object stored in the store
+message PatternRecognizer {
+    // The name of the recognizer, unique
+    string name = 1;
+    // The Regex used by this recognizer
+    string pattern = 2;
+    // The confidence of the result
+    float strength = 3;
+    // The entity name which this recognizer can detect
+    string entity = 4;
+    // The supported language
+    string language = 5;
```

Comment:
Let's keep all API consistent. either language or languageCode - I prefer language and in the comment write languageCode according to ISO XXX (as specified in the change Itye made)